_posts/en/meetings/2016-07-21-meeting.md

```diff
@@ -38,7 +38,7 @@ RC1 got some [feedback](https://github.com/bitcoin/bitcoin/issues/8383) noticing
 
 A common complaint is the lack of a way to export the HD seed. Jonasschnelli has a [pull request][#8206] which is easy to review and has a low impact that exports the seed to dumpwallet. Importing is a different issue with much more impact as the wallet currently doesn't support multiple seeds. This is a feature for 0.14.
 
-Luke-jr notes the default policy isn't performing as well as inadvisable policies for which he made a [pull request][#8388]. This is a pretty large change and requires some more discussion.
+Luke-jr notes the new default policy of using blockmaxweight isn't performing as well as using blockmaxsize in the current environment. He made a [pull request][#8388] to change this. This is a pretty large change and requires some more discussion.
```

Comment:
Release notes read:  
-For optimal runtime performance, miners using this release should specify
 -`-blockmaxweight` on the command line, and not specify `-blockmaxsize`.
 -Additionally (or only) specifying `-blockmaxsize`, or relying on default
 -settings for both, may result in performance degradation, as the logic to
 -support `-blockmaxsize` performs additional computation to ensure that
 -constraint is met.

I read that as it being recommended to use blockmaxweight.